### DISCUSSION TITLE
使用touchtone(lm-eval)测试Qwen2.5-3B-Instruct时， truthfulqa_mc2性能偏高怎么回事？

### DISCUSSION THREAD

**Discussion by mathCrazyy** [QwenLM/Qwen3#1127] [Q&A]
![image](https://github.com/user-attachments/assets/c2d5718a-6aa2-4e89-9955-df04fd03f443)
检查模型model-00002-of-00002.safetensors 的md5 fba2c2ad0cdbf6633b73ce2dc0ec0455

**Comment by mathCrazyy** [comment]
官网公布的数据 
![image](https://github.com/user-attachments/assets/20ba0750-ed64-465a-b19a-9d402872ab0b)

**Comment by jklj077** [comment]
The section at <https://qwenlm.github.io/blog/qwen2.5-llm/#qwen25-05b15b3b-performance> is about the base model. 
You were evaulating the instruct the model. 

I don't think we have provided results of the instruct model on TruthfulQA @hzhwcmhf

**Comment by mathCrazyy** [comment]
thanks for your help
